packages/inspection-capture-web/src/PhotoCapture/PhotoCapture.tsx

```diff
@@ -157,7 +161,13 @@ export function PhotoCapture({
     uploadQueue,
     tasksBySight,
   });
-  const handleOpenGallery = () => setCurrentScreen(PhotoCaptureScreen.GALLERY);
+  const handleOpenGallery = () => {
+    setCurrentScreen(PhotoCaptureScreen.GALLERY);
+    analytics.trackEvent('Gallery Opened', {
+      inspectionId,
+      category: 'gallery_opened',
```

Comment:
What is the point of `category` again? I forgot

Reply:
It was used in the first version/iteration, but now useless, I'll remove it.